File: Programmers/Lv_1/가운데글자가져오기.cpp
#include <iostream>
#include <string>
#include <vector>
using namespace std;

string solution(string s) {
    string answer = "";

    if (s.size() % 2 == 1) {
        answer += s[s.size() / 2];
    }
    else {
        answer += s[s.size() / 2 -1];
        answer += s[s.size() / 2];
    }

    return answer;
}

int main() {
    ios_base::sync_with_stdio(0);
    cin.tie(0);

    string s;
    cin >> s;

    cout << solution(s);

    return 0;
}

File: Programmers/Lv_1/가운데글자가져오기.py
def solution(s):
    answer = ''
    mid = len(s) // 2
    if len(s) % 2 == 0:
        answer += s[mid-1 : mid+1]
    else:
        answer += s[mid]
    
    return answer

def solution2(s):
    return str[(len(s)-1)//2 : len(s)//2+1]